Assets/Scenes/ZoomSystemTestScene.unity

```diff
@@ -361,7 +361,7 @@ MonoBehaviour:
       m_PreInfinity: 2
       m_PostInfinity: 2
       m_RotationOrder: 4
-  m_CustomBlends: {fileID: 0}
+  m_CustomBlends: {fileID: 11400000, guid: d59371a748f7acb40bb306d10bfbea1f, type: 2}
```

Comment:
Attaching the blends file to the camera so that we didn't forget about it in the future even though the 3D bit isn't in this scene

Reply:
I'm not sure of the implications of this but I'll trust you!